tools/windows-node-installer/pkg/client/kubeclient.go

```diff
@@ -0,0 +1,24 @@
+package client
+
+import (
+	"fmt"
+	"log"
+
+	client "github.com/openshift/client-go/config/clientset/versioned"
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
Is there any reason we are looking at the master build.? Currently they recommend to use v12.0.0 tag 
https://github.com/kubernetes/client-go

Reply:
awesome comment. I like how you noticed this. There are some functionalities not supported and we are using go.mod. Since the EC2 is not included in this PR, I did not fix on the version. If we are fixing the version we would have to go for this  
```suggestion
	"k8s.io/client-go/tools/clientcmd"https://github.com/kubernetes/client-go/issues/551#issuecomment-475465546
```